python/tvm/relay/frontend/tensorflow.py

```diff
@@ -406,8 +425,10 @@ def _impl(inputs, attr, params):
         # NHWC
         attr['layout'] = 'NHWC'
 
+        # Ignore the new attribute `half_pixel_centers` from TF2.0, for now.
+        #   The semantics of the attribute is not very clear at this point.
```

Comment:
maybe remove this line or add TODO?

Reply:
Just for the record: The comment refers to the lack of TF documentation to understand the intent of the attribute. The source code gives a better idea, but still...